scripts/deploy.sh

```diff
@@ -48,8 +49,13 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   if [ -n "${DATA_DISK_DEVICE}" ]; then
     export BOOTSTRAP_OPTS="${BOOTSTRAP_OPTS} bootstrap_host_data_disk_device=${DATA_DISK_DEVICE}"
   fi
+
   # force the deployment of haproxy for an AIO
   export DEPLOY_HAPROXY="yes"
+
+  # Enable host security hardening for AIO builds
+  export DEPLOY_HARDENING="yes"
```

Comment:
My understanding is that we need to carry this playbook because OSA have rejected backporting this feature to Liberty. If it's being backported we don't need this patch.